2-mandatory/7-eligible-students.js

```diff
@@ -14,10 +14,18 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendances) {
+  let arr = [];
+  for (let students of attendances) {
```

Comment:
You can use `const` here. `const` should be the default for declaring variables. Only if you know you'll be modifying later should you use `let`.